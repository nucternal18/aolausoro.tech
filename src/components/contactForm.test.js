import React from "react";
import axios from "axios";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import ContactForm from "./contactForm";

configure({ adapter: new Adapter() });

jest.mock("axios");

describe("test contact form", () => {
  test("Sends message when button is clicked", () => {
    const component = shallow(<ContactForm />);

    expect(component.length).toEqual(1);
  });

  test("expect to render contact form component", () => {
    const component = shallow(<ContactForm />);

    expect(component).toMatchSnapshot();
  });
});
