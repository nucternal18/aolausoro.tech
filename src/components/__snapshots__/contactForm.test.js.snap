// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test contact form expect to render contact form component 1`] = `ShallowWrapper {}`;
